video/out/w32_common.c

```diff
@@ -822,6 +840,10 @@ static void update_window_style(struct vo_w32_state *w32)
     const DWORD style = GetWindowLongPtrW(w32->window, GWL_STYLE);
     SetWindowLongPtrW(w32->window, GWL_STYLE, update_style(w32, style));
     w32->windowrc = wr;
+
+    w32->top_border_reduce = (w32->opts->border && !w32->opts->title_bar &&
+                              !w32->current_fs)
+                                ? get_invisible_border_size(w32) : 0;
```

Comment:
```suggestion
   w32->top_border_reduce = (w32->opts->border && !w32->opts->title_bar &&
                              !w32->current_fs) ? get_invisible_border_size(w32) : 0;
```
I dunno. Looks better imo. 85 characters isn't that long.

Reply:
changed